docs/enterprise/console-settings.yaml

```diff
@@ -256,7 +256,7 @@ settings:
              ::::
              :::::
 
-          1. After you click **Submit**, the modal presents the Java Web Token (**JWT**) for the service account. Temporarily save it somewhere secure, as you will not be able to view it again:
+          1. After you click **Submit**, the modal presents the Json Web Token (**JWT**) for the service account. Temporarily save it somewhere secure, as you will not be able to view it again:
```

Comment:
Just a note that Json seems to be one of those things that are in all caps based on my google search, but I don't have a pref either way.